examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -248,6 +254,25 @@ private static void runFullDemo(InetSocketAddress hostAddr, InetSocketAddress om
         // Start the microservice
         clients[0] = createTwitterManager(oms, microServiceName);
 
+        /* TODO: quinton: Fix this hack.
+         *
+         * For some reason, when adding a ConsensusRSMPolicy after the AtLeastOnceRPCPolicy
+         * it seems that the TwitterManager stub is returned before the TwitterManager microservice
+         * has been properly initialized.
```

Comment:
This was due to issue #792

Reply:
Great, thanks.  I'll rebase now that #792 is fixed, and retest.

Reply:
Rebased, removed this hack, and checked that all tests and examples pass.  Success!